community/samples/serving/helloworld-java-quarkus/README.md

```diff
@@ -0,0 +1,452 @@
+---
```

Comment:
Delete lines 1-9 -> the _index.md file "includes" this readme during the build and the frontmatter nor the readfile statement should be here

Reply:
removed